config/examples/delta/Anycubic/Predator/Configuration_adv.h

```diff
@@ -1524,6 +1524,9 @@
   //
   #define FSMC_UPSCALE 3
 
+  #define LCD_FULL_PIXEL_WIDTH 480
+  #define LCD_FULL_PIXEL_HEIGHT 320
```

Comment:
These should not be needed with the updates in MarlinFirmware/Marlin#18903.

Reply:
The predator uses a FSMC_GRAPHICAL_TFT not the serial display Anycubic uses for the i3 (and for the Chiron?). I couldn't find anything in the referenced PR that would affect that. (Having less config would of course be nice)

Reply:
The values given here for `LCD_FULL_PIXEL_WIDTH`/`HEIGHT` should now come out as a consequence of `FSMC_UPSCALE` without these lines being needed. But I will do a test to make sure.

Reply:
Found it scaled_tft.h is taking care of it now. Awesome!